tests/utils/test_replay_buffer.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+import pytest
+from gym.spaces import Box
+
+from src.utils.replay_buffer import ReplayBuffer
+
+spaces = {
+    "space1": Box(-1, 1, (3,)),
+    "space2": Box(-np.inf, 1, (3,)),
+    "space3": Box(-1, np.inf, (3,)),
+    "space4": Box(-np.inf, np.inf, (3,)),
```

Comment:
shapeが2次元以上の場合もテストしてください。例えば
```suggestion
    "space4": Box(-np.inf, np.inf, (3, 4, 5)),
```